src/time/location.cr

```diff
@@ -61,11 +61,11 @@ class Time::Location
     end
 
     def inspect(io : IO)
-      io << "Time::Zone<"
+      io << "Time::Zone("
       io << offset
       io << ", " << name
```

Comment:
Since we are here, they can be merged to `io << "Time::Zone" << offset << ", " << name`

Reply:
@j8r I don't think that would be an improvement.